assets/css/_ui_kit.scss

```diff
@@ -254,6 +254,26 @@ $color-tooltip-background: #333333;
   }
 }
 
+@mixin button-primary-v2 {
```

Comment:
There already is an existing button-primary button with different styles. I was inclined to introduce this as a separate mixin for now, but could alternatively update the existing primary buttons if desired.

Reply:
I think we should try and find a way to make this a single style if possible, or at least have a ticket for it?

Reply:
Just checked and the old style is only actually used in one place, confirmed with design that we can make the switch to the new styles for that location. Udpate incoming!

Reply:
Oops I missed one existing reference on the shuttle page, checking with design about that, but otherwise updated this new mixin to be `button-primary` and marked the old one as `button-primary-legacy`

Reply:
Confirmed with design to leave the shuttle route picker using the `button-primary-legacy` styles for now